README.md

```diff
@@ -7,6 +7,7 @@ See some of the projects we've built:
 | Project                                             | Technologies                     | Description                                                             | Deployment                                                                                 |
 | --------------------------------------------------- | -------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
 | [angular-apollo-tailwind](/angular-apollo-tailwind) | Angular v13, Apollo, TailwindCSS | A GitHub client built with Angular v13, TailwindCSS, and Apollo Client. | [https://angular-apollo-tailwind.starter.dev](https://angular-apollo-tailwind.starter.dev) |
+| [next-react-query-tailwind](/next-react-query-tailwind) | NextJS, React Query, Tailwind. | [https://next-react-query-tailwind.starter.dev](https://next-react-query-tailwind.starter.dev) |
```

Comment:
This doesn't quite match the line above. We have a "Description" column. I think we can sunset that for the technologies column though if you wanted to consolidate.